src/Board.cpp

```diff
@@ -14,4 +19,51 @@ Board::Board(int rowNumber,
     board_.assign(rowNumber_, singleRow);
 }
 
+int getRowNumberFromFile(std::string path)
+{
+    std::ifstream inputFile(path);
+    json j;
+    inputFile >> j;
+    return j["rowNumber"];
+}
+
+int getColNumberFromFile(std::string path)
+{
+    std::ifstream inputFile(path);
+    json j;
+    inputFile >> j;
+    return j["colNumber"];
+}
+
+Board::Board(std::string path)
+{
+    JSONFileReader fileReader(path);
+
+    if ((fileReader.readRowsNumber() < 3) or (fileReader.readColsNumber() < 3))
```

Comment:
This logic with checks should be moved just into JSONFileReader.
Also, getCol/RowNumberFromFile can be removed because they are already in JSONFileReader.